File: VB/Q147080/Tab/ViewInfo/MySkinTabHeaderViewInfo.vb
Imports DevExpress.XtraTab.ViewInfo

Namespace DxSample.Tab.ViewInfo
    Public Class MySkinTabHeaderViewInfo
        Inherits SkinTabHeaderViewInfo

        Public Sub New(ByVal viewInfo As BaseTabControlViewInfo)
            MyBase.New(viewInfo)
        End Sub

        Private rows_Renamed As New MyTabRowViewInfoCollection()
        Public Overrides ReadOnly Property Rows() As BaseTabRowViewInfoCollection
            Get
                Return rows_Renamed
            End Get
        End Property
        Protected Overrides Function UpdateFillPageBounds() As Boolean
            Return False
        End Function
    End Class
End Namespace

File: CS/Q147080/Tab/ViewInfo/MySkinTabHeaderViewInfo.cs
using DevExpress.XtraTab.ViewInfo;

namespace DxSample.Tab.ViewInfo {
    public class MySkinTabHeaderViewInfo : SkinTabHeaderViewInfo {
        public MySkinTabHeaderViewInfo(BaseTabControlViewInfo viewInfo) : base(viewInfo) { }
        private MyTabRowViewInfoCollection rows = new MyTabRowViewInfoCollection();
        public override BaseTabRowViewInfoCollection Rows { get { return rows; } }
        protected override bool UpdateFillPageBounds() { return false; }
    }
}
